houses/forms.py

```diff
@@ -7,3 +8,10 @@ class HouseForm(forms.ModelForm):
     class Meta:
         model = Fraternity
         fields = '__all__'
+
+    def clean_calendar_link(self):
+        data = self.cleaned_data['calendar_link']
+        if not re.match("^https://calendar.google.com/calendar/embed\?src=(.*)", data):
```

Comment:
@tmwbook does this address your concerns?

Reply:
I am in the process of writing unit tests to see if this is verbose enough.